src/phoenix/experimental/evals/templates/template.py

```diff
@@ -16,101 +15,127 @@
 
 
 @dataclass
-class PromptTemplate:
+class PromptOptions:
+    provide_explanation: bool = False
+
+
+class ClassificationTemplate(ABC):
```

Comment:
I'm pretty confused here: why did we nix PromptTemplate? I think what you are getting at is that there is no base template but on first glance it does make the type signatures kinda confusing.